internal/host/transition.go

```diff
@@ -685,6 +686,17 @@ func (th *transitionHandler) PostRefreshHost(reason string) stateswitch.PostTran
 			reason == statusInfoInstallationInProgressTimedOut {
 			template = statusInfoInstallationInProgressWritingImageToDiskTimedOut
 		}
+		if reason == statusInfoPreparationTimeout {
```

Comment:
I don't think we should rely on status info to make logical decisions.

Reply:
I will change this